src/TextHandler.mss

```diff
@@ -88,7 +95,7 @@ function FiguredBassTextHandler (this, textObject) {
 }  //$end
 
 
-function AddFormattedText (parentElement, textObject) {
+function AddFormattedText (self, parentElement, textObject) {
```

Comment:
The addition of `self` here is a bit funny, since it's not used?

Reply:
This is because I moved to "Template Actions", and `AddFormattedText` now became the Template Action method of the `FormattedText` placeholder object. Methods always get a "self" as first argument, but it's of no use here.